UI/src/view/point_asset/trafficSignLabel.js

```diff
@@ -93,23 +177,35 @@
         return labelProperty && labelProperty.convertion ? labelProperty.convertion.call(trafficSign) : trafficSign.value;
       }
 
+      function getAdditionalInfo(){
+        return labelProperty && labelProperty.isToShowAdditionalInfo ? labelProperty.isToShowAdditionalInfo.call(trafficSign) : '';
+      }
+
       function getUnit() {
         return labelProperty && labelProperty.unit ? labelProperty.unit.call(trafficSign) : '';
       }
 
+      function getMaxLength() {
+        return labelProperty && labelProperty.maxLabelLength ? labelProperty.maxLabelLength : 20;
```

Comment:
This method shouldn't return a fixed value "20" , should return undefined or "0"

Reply:
According to the user story, the maximum length to show strings on the label is 20.
For that reason, you add 50 to the maxLabelLength property.
In the future, we can review this method and test all cases again, but for now, it's working fine.